source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -67,6 +67,45 @@ void BioXASAppController::shutdown()
 	CLSAppController::shutdown();
 }
 
+void BioXASAppController::applyStylesheets()
+{
+	// Go through list of stylesheets to be applied,
+	// composing a 'master' sheet.
+
+	QString stylesheet;
+
+	// AMToolButton
+
+	QFile qss1(":/AMToolButton.qss");
+
+	if (qss1.open(QFile::ReadOnly))
+		stylesheet.append(QString("\n\n%1").arg(QLatin1String(qss1.readAll())));
+
+	qss1.close();
+
+	// AMDeadTimeButton
+
+	QFile qss2(":/AMDeadTimeButton.qss");
```

Comment:
Unfortunately, the stylesheet needs to be applied all at once at the AppController level; subsequent calls to `qApp->setStylesheet(...)` overwrite previous entries. We could do something like a general `AMDatamanAppController::getStylesheet(...)` that could be reimplemented for each application as desired--that's an idea.